include/Dialects/LinalgTransform/LinalgTransformOps.td

```diff
@@ -166,6 +166,44 @@ def LowerToLLVMOp : Linalg_Transform_Operation<"lower_to_llvm"> {
   let assemblyFormat = "attr-dict";
 }
 
+def GetParentLoopOp : Linalg_Transform_Operation<"get_parent_loop"> {
+  let description = [{Obtains a handle to a parent loop of the given
+  operation.}];
+
+  let arguments = (ins PDL_Operation:$target,
+                   DefaultValuedAttr<Confined<I64Attr, [IntPositive]>,
+                                     "1">:$num_loops);
+  let results = (outs PDL_Operation:$transformed);
+
+  let assemblyFormat = "$target attr-dict";
+}
+
+def UnrollLoopOp : Linalg_Transform_Operation<"unroll_loop"> {
+  let description = [{Unrolls the given loop with the given unroll factor.}];
+
+  let arguments = (ins PDL_Operation:$target,
+                   Confined<I64Attr, [IntPositive]>:$factor);
+  
+  let assemblyFormat = "$target attr-dict";
+}
+
+def PipelineLoopOp : Linalg_Transform_Operation<"pipeline_loop"> {
+  let arguments = (ins PDL_Operation:$target,
+                   DefaultValuedAttr<I64Attr, "1">:$modulo,
```

Comment:
This is not up to date anymore, we now have:
```
    Option<"II", "II", "unsigned", /*default=*/"1", "Iteration Interval.">,
    Option<"readLatency", "read-latency", "unsigned", /*default=*/"1",
```

@ThomasRaoux I believe I took the term "Iteration Interval" from your terminology.
I assume it is the same thing as the traditional "initiation interval" ?

Reply:
Thanks. I would strongly suggest to give things more meaningful names than "II".

Reply:
`iteration_interval` seems fine here, the old one can then go away.

Reply:
initiation interval is probably the right type but `iteration_interval` is fine as well.